src/core/tracking.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-underscore-dangle */
+
+import config from 'config';
+import log from 'core/logger';
+
+
+export class Tracking {
+
+  constructor({trackingId, trackingEnabled, _log = log} = {}) {
```

Comment:
I think destructuring should have spaces so `constructor({ trackingId, trackingEnabled, _log = log } = {}) {`